e2e-tests/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 80,
```

Comment:
are we consistent with other repos?

Reply:
Ah, good catch (this was left over from the previous PR). I will update to use:
```
"@matterlabs/eslint-config-vue": "^1.1.3",
"@matterlabs/prettier-config": "^1.0.2",
```